webtau-core/src/main/java/com/twosigma/webtau/data/render/ToStringRenderer.java

```diff
@@ -19,6 +19,8 @@
 public class ToStringRenderer implements DataRenderer {
     @Override
     public String render(Object data) {
-        return data.toString();
+        return data instanceof String ?
```

Comment:
The functionality of this renderer has changed a bit I'd say. It's not really ToStringRenderer but just StringRenderer.

Reply:
good catch. Renaming